### DISCUSSION TITLE
Can I import `postcss`?

### DISCUSSION THREAD

**Discussion by stabldev** [django-components/django-components#336] [Q&A] [answered]
Is it possible to add `postcss` like this? or is there any way that I can import postcss?
```py
@component.register("text_editor")
class TextEditor(component.Component):
    template_name = "text_editor/text_editor.html"

    class Media:
        css = "text_editor/text_editor.postcss"
```

**Comment by anups1ngh** [comment, marked as answer]
Yes, you can use PostCSS in your Django project by integrating it into your build process. The `css` attribute in the `Media` class of your Django component is typically used for including stylesheets directly, but it doesn't have built-in support for PostCSS.

To use PostCSS, you'll need to set up a build process that runs PostCSS on your stylesheets before they are included in your Django project. Here's a general outline of the steps you can take:

1. **Install PostCSS and required plugins:**
   ```
   npm install postcss postcss-cli postcss-preset-env
   ```

2. **Create a PostCSS configuration file:**
   Create a `postcss.config.js` file in your project's root directory:
   ```javascript
   // postcss.config.js
   module.exports = {
     plugins: {
       'postcss-preset-env': {},
       // Add other PostCSS plugins as needed
     },
   };
   ```

3. **Update your build process:**
   Modify your build process (e.g., using npm scripts or a task runner like Gulp) to run PostCSS on your stylesheets. For example, if you have a `build` script in your `package.json`:
   ```json
   "scripts": {
     "build": "postcss path/to/your/styles.css -o path/to/your/output.css"
   }
   ```

4. **Include the processed CSS in your Django component:**
   After running the build process, include the processed CSS in your Django component:
   ```python
   @component.register("text_editor")
   class TextEditor(component.Component):
       template_name = "text_editor/text_editor.html"

       class Media:
           css = {
               'all': ('path/to/your/output.css',),
           }
   ```

Remember to replace `'path/to/your/styles.css'` and `'path/to/your/output.css'` with the actual paths in your project.

This way, you can leverage PostCSS in your Django project by incorporating it into your build pipeline.

Mark it as answer if this helps !

**Reply by stabldev** [reply]
Okay so I need to build and config output css...
Yes looks good! thanks @anups1ngh 
> Mark it as answer if this helps !

Sure :+1:
